### PR TITLE
Done PHEE-238 Added test case for mockserver stub

### PR DESCRIPTION
**Pull request by danishjamal104**
## Changes in this PR
1. New scenario implemented `Mockserver stub test` under `mockserver.feature`.
2. This test case will test the working of mockserver.

Refer jira ticket >> [PHEE-238](https://fynarfin.atlassian.net/browse/PHEE-238)

[PHEE-238]: https://fynarfin.atlassian.net/browse/PHEE-238?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/test/java/org/mifos/integrationtest/config/MockServerConfig.java [hunk 1]
```diff
@@ -23,5 +23,10 @@ public WireMockServer getMockServer() {
         return MockServerConfig.singleInstance;
     }
 
+    @Override
+    public String getBaseUri() {
+        return "http://localhost:"+getMockServer().port();
```
Comment (fynmanoj) [inline src/test/java/org/mifos/integrationtest/config/MockServerConfig.java:28]: can we use mock server at localhost ?

**Review by fynmanoj — CHANGES_REQUESTED** [review]
clarrification:

**Comment by danishjamal104** [comment]
Yes

File: src/test/java/org/mifos/integrationtest/config/MockServerConfig.java (hunk 1 above)
Comment (danishjamal104) [reply src/test/java/org/mifos/integrationtest/config/MockServerConfig.java:28]: yes

Comment (fynmanoj) [reply src/test/java/org/mifos/integrationtest/config/MockServerConfig.java:28]: what is the use of adding a server in localhost? what are we trying to test here?

Comment (danishjamal104) [reply src/test/java/org/mifos/integrationtest/config/MockServerConfig.java:28]: testing if mockserver is able to start a test or not.

**Pull request merged**